paper-tabs.html

```diff
@@ -160,28 +159,28 @@
 
   <template>
 
-    <div class-name="[[scrollButtonClass]]">
-      <core-icon icon="paper-tabs:chevron-left" class-name="[[leftIconClass]]"
+    <div class$="[[computeScrollButtonClass(scrollable, hideScrollButton)]]">
```

Comment:
This is interesting. So, since the computed method is declared inline, a property isn't created to receive the result. That seems like a good thing.